docs/user/03-traces.md

```diff
@@ -461,62 +461,70 @@ System-related spans reported by Istio are filtered out without the opt-out opti
 
 ## Troubleshooting
 
-### Traces Not Arriving at the Destination
+### No Spans Arrive at the Backend
 
-   Cause: Wrong backend endpoint configuration or authentication credentials are used.
+Cause: Incorrect backend endpoint configuration (e.g., using the wrong authentication credentials) or the backend being unreachable.
 
-   Remedy: Investigate the cause with the following steps:
-   1. Check the `telemetry-trace-collector` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`.
+Remedy: 
+- Check the `telemetry-trace-collector` Pods for error logs by calling `kubectl logs -n kyma-system {POD_NAME}`.
+- Check if the backend is up and reachable.
 
-### Custom Spans don't Arrive at the Destination, but Istio Spans do
+### Not All Spans Arrive at the Backend
 
-   Cause: Your SDK version is incompatible with the OTel collector version.
+Cause: The backend is reachable and the connection is properly configured, but some spans are refused.
```

Comment:
That's a symptom, not a cause. Is the cause "The backend export rate is too low compared to the gateway ingestion rate."  ?

Reply:
Agree, changed to a Symptom